### PR TITLE
added a translate command

### PR DESCRIPTION
**Pull request by aaj2005**

### CODE REVIEW

File: cogs/commands/misc.py [hunk 1]
```diff
@@ -143,6 +143,17 @@ async def chat(self, ctx: Context, message: str = None, gpt4: bool = False):
     async def gpt4(self, ctx: Context, message: str):
         await self.chat(ctx, message, True)
 
+    @commands.hybrid_command()
+    async def translate(
+        self, ctx: Context, source: str, target: str, message: clean_content
```
Comment (Mole1424) [inline cogs/commands/misc.py:148]: add a help text to explain what source and target are (see [https://github.com/UWCS/apollo/blob/4d30f637eb5c0ba27180ef552ab5999c1a7c45f5/cogs/commands/say.py#L15](https://github.com/UWCS/apollo/blob/4d30f637eb5c0ba27180ef552ab5999c1a7c45f5/cogs/commands/say.py#L15))

Also might be an idea to set a deafult value of source to en or auto

File: cogs/commands/misc.py [hunk 2]
```diff
@@ -143,6 +143,17 @@ async def chat(self, ctx: Context, message: str = None, gpt4: bool = False):
...
+    ):
+        translated = GoogleTranslator(source=source, target=target).translate(message)
+        await ctx.send(translated)
+
+    @commands.hybrid_command()
+    async def joeltech(self, ctx: Context):
```
Comment (Mole1424) [inline cogs/commands/misc.py:154]: tbh im not entirely sure if we need this given :joel_tech: can already be written

**Review by Mole1424 — COMMENTED** [review]
some changes needed

File: cogs/commands/misc.py (hunk 2 above)
Comment (adamskrz) [reply cogs/commands/misc.py:154]: the command should be Joel not Joel tech for this to be effective

**Comment by Mole1424** [comment]
use of camelCase in pytho brings me pain but fine

**Pull request merged**